### ISSUE TITLE
[CHECK] check failed when loading a normal url from chrome web UI pages(chrome:// pages)

### ISSUE THREAD

**Issue by srirama179** [Samsung/Castanets#191]
_Check fails in **content::NavigatorImpl::CheckWebUIRendererDoesNotDisplayNormalURL()** when loading a normal URL after loading a chrome internal URLs._

**Steps to reproduce the issue**
1. Load chrome://version
2. Navigate to google.com

or 

1. Delete chromium folder in .config folder in home directory (this will start chrome browser fresh)
2. Start browser process without url (it will load _chrome://welcome/?variant=everywhere_)
3. Navigate to google.com

**Comment by srirama179** [comment]
Skipping the CHECK works fine without any further issues.
I believe we use (process per tab model?) same process for all the URLs, so just allowing WebUI and normal URL in the same tab for castanets should be fine.
So is adding a castanets check in content_browser_client and returning true good?

**Comment by Sunwoo-Nam** [comment]
Hello, Srirama.
I guess |IsURLAcceptableForWebUI| returns 'true' in 'Chrome' only.
In the Chromium opensource, it is always 'false' regardless of Castanets.
So we can just keep this crash issue. This is just my opinion.. :)
We need to listen to other opinions.

**Comment by srirama179** [comment]
You are right, chromium also returns false but there is no crash in chromium because the _webui_bindings_ flag is also false. As I debugged further, in chromium the process model used is process-per-site-instance (default process model) and the process id keeps changing when I load chrome://version, google,com, yahoo.com in the same tab.
Where as in our case the process id is same for the above 3 URLs which may be the reason for the bindings check and the crash.
As you said we can wait for opinion from others.
